routes/tickets.js

```diff
@@ -85,8 +85,10 @@ module.exports = function (app, passport) {
             })
     })
 
-    app.get('/tickets', onlySuperuser, (req, res) => {
-        Ticket.findAll({include: [TicketUpdate, User]})
+    app.get('/tickets', (req, res) => {
+        var where = req.user.isSuperuser ? {} : {userId:req.user.id}
+        console.log(where);
```

Comment:
- Better check for `req.user` first and return 401 explicitly rather than 500 implicitly
 - I'm guessing that `console.log(where)` slipped away?